.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: required
 
-services:
-  - docker
+-services:
+ -  - docker
 
 script:
-  - docker-compose run async --quiet
+  - docker-compose run diff-sync
```

Comment:
Now this ok

Reply:
Actually when I made last changes test.sh doesn't find `git` command
<img width="762" alt="errt" src="https://user-images.githubusercontent.com/10897901/32730592-e2e9ca66-c887-11e7-8154-acf3ac0d2248.png">

Reply:
hmmmm, yes, I understand... I need to push your changes of `Dockerfile` to the dockerhub...